### PR TITLE
Update ReadMe to include CocoaPods info

### PR DESCRIPTION
**Pull request by rebello95**
- Update README swift-protobuf version
- Add CocoaPods info to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,35 @@ Swift Package Manager builds may also be made on Linux
...
+## Manual integration
+
+When not using CocoaPods, Swift gRPC includes **vendored copies** of the
+gRPC Core library and BoringSSL (an OpenSSL fork that is used by
+the gRPC Core). These are built automatically in Swift Package
 Manager builds.
 
+After [building your project](#building-your-project), add the generated
+`SwiftGRPC.xcodeproj` to your project, and add build dependencies
+on **BoringSSL**, **CgRPC**, **gRPC**, and **CzLib**.
```
Comment (MrMage) [inline README.md]: I think the zlib dependencies is obsolete, and `gRPC` is now called `SwiftGRPC`.

File: README.md [hunk 2]
```diff
@@ -101,12 +115,12 @@ For an example of this in Swift, please see the
 
 ## Having build problems?
 
-grpc-swift depends on Swift, Xcode, and swift-proto. We are currently
+grpc-swift depends on Swift, Xcode, and swift-protobuf. We are currently
 testing with the following versions:
 
 - Xcode 9.1
 - Swift 4.0-dev
```
Comment (MrMage) [inline README.md]: Could you change this to „Swift 4.0“ while at it?

File: README.md (hunk 1 above)
Comment (rebello95) [reply README.md]: Good catch (copy/paste). Updated here: https://github.com/grpc/grpc-swift/pull/216

File: README.md (hunk 2 above)
Comment (rebello95) [reply README.md]: Updated in https://github.com/grpc/grpc-swift/pull/216

**Pull request merged**